### DISCUSSION TITLE
Custom delay page in Firefox >57?

### DISCUSSION THREAD

**Discussion by BigLesbianFox** [proginosko/LeechBlockNG#375] [Ideas]
I know it was possible to set a custom delay page in the pre-WebExtension days, but this feature appears to be no more (the relevant FAQ page that directed you to set `<html id="leechblockDelayingPage">` is gone.)

Would it be possible to specify a static URL to be loaded in lieu of the default delay page, the same way that custom block pages are currently handled, and then simply tell LeechBlock to redirect from there after a timeout? Googling for "firefox extension redirect" produces some results that suggests this is possible from within the WebExtension API, but I'm not certain.

**Comment by xuiqzy** [comment]
If anyone is looking for the specific use case of hiding the countdown timer, I made the [Suggestion: Option to hide countdown timer on delayed page](https://github.com/proginosko/LeechBlockNG/discussions/397).

**Comment by proginosko** [comment]
Support for custom delaying page added in version 1.7. Documentation will follow, but for now see [here](https://github.com/proginosko/LeechBlockNG/discussions/421#discussioncomment-13382568).
